src/Hamiltonians/HubbardReal1DEP.jl

```diff
@@ -90,3 +99,12 @@ function get_offdiagonal(h::HubbardReal1DEP, address::SingleComponentFockAddress
     naddress, onproduct = hopnextneighbour(address, chosen)
     return naddress, - h.t * onproduct
 end
+
+function get_offdiagonal(h::HubbardReal1DEP{<:Complex}, add::SingleComponentFockAddress, chosen)
+    naddress, onproduct = hopnextneighbour(add, chosen)
+    if chosen % 2 == 0
+        return naddress, - conj(h.t) * onproduct
+    else
+        return naddress, - h.t * onproduct
+    end
+end
```

Comment:
ditto.

Maybe this should call a function `_get_offdiagonal_hubbard_real_1D` that is only implemented once for the three models.